BaseTools/Source/C/GenFw/GenFw.c

```diff
@@ -443,6 +443,64 @@ Routine Description:
   return STATUS_SUCCESS;
 }
 
+// MU_CHANGE begin

```

Comment:
Looks like most functions local to the file are `STATIC`. Can you add that as well for consistency?

Reply:
Added. Confirmed no issue building basetools or QemuQ35Pkg with the bins.